rebar.config.script

```diff
@@ -21,7 +21,8 @@ Rebar2Deps = [{idna, ".*",
 
               {ssl_verify_hostname, ".*",
                {git, "https://github.com/deadtrickster/ssl_verify_hostname.erl",
-                {tag, "1.0.5"}}}],
+                {tag, "1.0.5"}}},
+              {cowboy, ".*", {git, "https://github.com/ninenines/cowboy.git", {tag, "1.0.4"}}}],
```

Comment:
maybe we should test the env here for example if a TEST has been set in the env. So we don't need to fetch cowboy all the time? Thoughts?